Bottom: fa78051214f840e13a44f2c4f364a11708d5f09e
Top:    b1268db2128e71c7a82feef2031351628c138872
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-04 16:20:56 +0800

Refresh of autofs-5.0.7-syncronize-handle_mounts-shutdown.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 7a5a08e..92287fb 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1488,7 +1488,7 @@ static void handle_mounts_finish(void)
 
 	finish_mutex_lock();
 	/* Poke signal handler */
-	pthread_kill(state_mach_thid, SIGTERM);
+	pthread_kill(state_mach_thid, SIGCONT);
 	error(LOGOPT_ANY, "before signal fc.busy %d", fc.busy);
 	fc.busy++;
 	finish_cond_wait();
